kiosk/src/Services/GamepadManager.ts

```diff
@@ -242,8 +242,7 @@ class GamepadManager {
             gamepad.axes &&
             gamepad.axes.length >= this.minAxisRequired &&
             gamepad.buttons &&
-            gamepad.buttons.length >= this.minButtonPinRequired &&
-            !!gamepad.mapping
```

Comment:
Why can this check get taken away?

Reply:
I borrowed this condition from gamepad filtering code I found in an internal React library developed by the Xbox team. I wasn't entirely sure of its purpose, and turns out doesn't suit our needs because it filters out shoebox controllers.